### PR TITLE
#524 Updated to jcabi-parent 0.32.1

### PR DESCRIPTION
**Pull request by carlosmiranda**

### CODE REVIEW

**Comment by davvd** [comment]
Many thanks, I will find someone to review it before we merge

**Comment by davvd** [comment]
@ggajos it's yours,please review

File: src/main/java/com/thindeck/life/Lifecycle.java [hunk 1]
```diff
@@ -53,6 +54,7 @@
  * @author Yegor Bugayenko (yegor@tpc2.com)
  * @version $Id$
  * @since 0.1
+ * @checkstyle ClassDataAbstractionCoupling (300 lines)
```
Comment (ggajos) [inline src/main/java/com/thindeck/life/Lifecycle.java]: @carlosmiranda let's add pdd so we can try to decrease coupling here

Comment (carlosmiranda) [inline src/main/java/com/thindeck/life/Lifecycle.java]: I disagree. I think that's unnecessary, the class is just 130 lines for now. The warning was introduced by the use of `ServletMfs` due to the deprecation of the static `Manifests.append` class; it does not make the class any less cohesive than before.

Comment (ggajos) [inline src/main/java/com/thindeck/life/Lifecycle.java]: ok, make sense

**Comment by ggajos** [comment]
@rultor merge please

**Comment by carlosmiranda** [comment]
@yegor256 Can you confirm?

**Comment by yegor256** [comment]
@rultor merge

**Comment by carlosmiranda** [comment]
@yegor256 Strange... it asked me again anyway. :)

**Comment by carlosmiranda** [comment]
@rultor merge

**Comment by davvd** [comment]
@ggajos Thank you, I have added **16 mins** to your account in payment/transaction "`AP-3J6775672Y0241824`"... extra minutes for review comments (c=1)... +16 to your rating, your total score is [+3064](http://www.netbout.com/b/35094?open=rating)

**Comment by davvd** [comment]
@rultor please deploy

**Pull request merged**